### PR TITLE
Updating sample app icons

### LINKED ISSUE #142: [Enhancement] Sample App Icon, Splash Screen, etc.
## Summary

Add a new app icon and splash screen to the Sample App

## API Changes

It may be a good idea to add an app icon and splash screen to the Sample App as more people add more samples going forward instead of using the blue Xamarin image for the app icon and the image within the app itself in the FlyoutHeader.  @sthewissen mentions in #114 to use a cool new logo.  If he is finalizing a new logo, perhaps we can use that for the app icon and the splash screen.  Plus, it will keep the sample app aligned with the general theme of the toolkit. Thoughts?

## Intended Use Case

App icon, splash screen
iOS and Android

### PR DESCRIPTION
**Pull request by adenearnshaw**
### Description of Change ###

Updated the Launch Icons and Splash pages for the iOS, Android and UWP Sample apps with the elected project logo.

### Bugs Fixed ###

- Fixes #142 

Provide links to issues here. Ensure that a GitHub issue was created for your feature or bug fix before sending PR.

### API Changes ###

None

Added: 
 
- SplashActivity.cs with associated Android style and background resource
- Android splash logo
- UWP Tile icons, generated by Manifest editor
- Masked version of logo used for UWP icons

Changed:

- Replaced Android Icon & Launcer images
- Replaced iOS Icon & Launcher Storyboard

### Behavioral Changes ###

None

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description) - N/A
- [ ] Has samples (if omitted, state reason in description) - N/A
- [X] Rebased on top of main at time of PR
- [X] Changes adhere to coding standard
- [ ] Updated documentation - N/A

  
5BC8FA03-2AF1-4147-B6B3-01E72554C213

### CODE REVIEW

**Review by Char0394 — APPROVED** [review]
Thanks for your contribution. Looks good :)

**Comment by adenearnshaw** [comment]
Thanks @Char0394 although, I think I've reset you're review when I rebased, sorry

**Review by Char0394 — APPROVED** [review]

File: XamarinCommunityToolkitSample.Android/SplashActivity.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+namespace Xamarin.CommunityToolkit.Sample.Droid
+{
+	[Activity(Label = "XamarinCommunityToolkitSample", Icon = "@mipmap/icon", Theme = "@style/SplashTheme", MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation)]
+	public class SplashActivity : AppCompatActivity
+	{
+		protected override void OnCreate(Bundle savedInstanceState)
+		{
+			base.OnCreate(savedInstanceState);
+			var intent = new Intent(this, typeof(MainActivity));
+			StartActivity(intent);
```
Comment (AndreiMisiukevich) [inline XamarinCommunityToolkitSample.Android/SplashActivity.cs:17]: ```StartActivity(typeof(MainActivity));``` ? But ugh...

**Review by AndreiMisiukevich — APPROVED** [review]

File: XamarinCommunityToolkitSample.Android/SplashActivity.cs (hunk 1 above)
Comment (adenearnshaw) [reply XamarinCommunityToolkitSample.Android/SplashActivity.cs:17]: I'll be more terse next time

Comment (AndreiMisiukevich) [reply XamarinCommunityToolkitSample.Android/SplashActivity.cs:17]: No worries :) That's just my *personal* preferences

**Pull request merged**